lib/modules/datasource/go/index.ts

```diff
@@ -48,7 +48,8 @@ export class GoDatasource extends Datasource {
   @cache({
     namespace: `datasource-${GoDatasource.id}`,
     // TODO: types (#22198)
-    key: ({ packageName }: Partial<DigestConfig>) => `${packageName}-digest`,
+    key: ({ packageName }: Partial<DigestConfig>) =>
+      `getReleases:${packageName}-digest`,
```

Comment:
```suggestion
    key: ({ packageName }: GetReleasesConfig) =>
      `getReleases:${packageName}-digest`,
```
it seems we use a totally wrong type here 🙈